tests/iob.test.js

```diff
@@ -677,18 +700,15 @@ describe('IOB', function() {
         };
 
         var hourLaterInputs = inputs;
-        hourLaterInputs.clock = moment('2016-06-13 00:30:00.000'); //new Date(now + (30 * 60 * 1000)).toISOString();
-        var hourLater = require('../lib/iob')(hourLaterInputs)[0];
+        hourLaterInputs.clock = moment('2016-06-13 00:30:00.000');
+        var hourLaterWithOverlap = require('../lib/iob')(hourLaterInputs)[0];
 
-        hourLater.iob.should.be.lessThan(0.5);
-        hourLater.iob.should.be.greaterThan(0.45);
+        hourLater.iob.should.be.greaterThan(hourLaterWithOverlap.iob);
```

Comment:
Is this correct?  Or should it be `>=` rather than `>`?

Reply:
It passes on my VM, so after digging into the iob code, I changed it to test greaterThan the original IOB - 0.5 to account for variations.